README.md

```diff
@@ -719,7 +743,10 @@ Ph.D. Thesis. California Inst. Of Technology, Pasadena, CA, 1993. [PDF](https://
 - <a name="Lichtsteiner06PhD"></a>Lichtsteiner, P.,  
 *[A temporal contrast vision sensor](https://doi.org/10.3929/ethz-a-005279479)*,  
 Ph.D. Thesis, ETH Zurich, Zurich, Switzerland, 2006. [PDF](http://www.ini.uzh.ch/~tobi/papers/lichtsteinerThesis2006.pdf)
-- <a name="Berner11PhD"></a>Berner, R.,  
+- <a name="Matolin10PhD"></a>Matolin, D.,
```

Comment:
formatting...

Reply:
Sorry but the formatting seems fine to me (nano)